v2/trace/aws/aws.go

```diff
@@ -0,0 +1,51 @@
+package awstrace
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/aws/aws-sdk-go/service/sqs"
+)
+
+// WrapSession will wrap the AWS session and
+// injecting tracing headers into the outgoing requests for these operations:
+// - sns.Publish
```

Comment:
we have "sns.Publish" in the doc, but then use "sns/Publish" later?

Reply:
We can change the handler operation name to use dot to match datadog/oc trace

Reply:
I think we are using / in the applications

Reply:
@rickardenglund Which applications?
I perfer dot in this case

Reply:
The applications being for example users and hierarchy. This results in the naming of spans right?

So I think this depends on whether you want it to blend in with the data dog stuff or the enlight stuff.

Either way I think "." is perfectly fine But it is nice to same in doc and code :)

Reply:
What about this solution, which highlight the functions it supports. Not which operations, which is not entirely the same